src/Servers/IIS/IIS/src/Core/WrappingStream.cs

```diff
@@ -107,6 +107,7 @@ public override int EndRead(IAsyncResult asyncResult)
         public override void EndWrite(IAsyncResult asyncResult)
             => _inner.EndWrite(asyncResult);
 
+        [Obsolete("This Remoting API is not supported and throws PlatformNotSupportedException.", DiagnosticId = "SYSLIB0010", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Why is `[Obsolete(...)]` appropriate here❔ Unless it works in some scenarios, keeping the method around doesn't accomplish anything except give those who rebuild a warning things will break when they run. We usually only `[Obsolete]` things that still work.

Reply:
The error message that accompanied this obsoletion mentioned to mark overrides as obsolete. Also:  https://github.com/dotnet/aspnetcore/pull/23978/#discussion_r457713142

Reply:
That makes sense if the `[Obsolete]` base works. I'm not understanding why the override exists any longer. Same question for https://github.com/dotnet/aspnetcore/pull/23978/files#diff-27ac84ac193b5e4debda40d50bfd8d52R110

Where were you linking other than back to this PR❔

Reply:
It was a comment by @ericstj explaining what was needed here.

Reply:
This stream is wrapping a stream and providing different behavior than base. If you removed the override it would be a behavior change.

Reply:
Suggest the comment should be reworded then. It flat-out states the API doesn't work while, from what you say @ericstj, everything will be fine i.e. this comment applies only to the overridden base.

Reply:
I'm talking about your API.  Base stream will still throw which is most likely what will happen, but passed in stream may override this and you should preserve calling that.  I don't think its important to customize the message.